### ISSUE TITLE
Will Fae move to SassC-Rails?  

### ISSUE THREAD

**Issue by asecondwill** [wearefine/fae#455]
It's triggering a warning in rails-s output 

```
 If you use Sass as a plug-in for a Ruby web framework, we recommend using the
  sassc gem: https://github.com/sass/sassc-ruby#readme
```

**Comment by jamesmk** [comment]
Thanks for raising this issue. It looks like we will have to migrate off of `sass-rails` as it's now EOL. The question is to migrate to `sassc` or something like webpack.

Both solutions have bigger implications and wouldn't be considered backwards compatible. So unfortunately this isn't a drop in replacement.

**Comment by jamesmk** [comment]
Fixed in #594
